### ISSUE TITLE
Cannot make the application working

### ISSUE THREAD

**Issue by SimoneMSR** [jcbvm/i18n-editor#40]
Hello,
I have used i18n-editor for a while with great pleasure. But now when I launch it I can just see the icon in the status-bar (and close the minimized icon with the 'close window button') , but I do not see any window.

I tried to completely uninstall (folder removed), reinstall, or launch it as a jar file. Nothing. What is happening?

Version: 1.0.0
Platform: Win 10
Java version 1.8.0_161

**Comment by jcbvm** [comment]
Looks to me like something is got wrong with java. 
I assume you did try a restart of your pc?

**Comment by SimoneMSR** [comment]
Yes, I have restarted the system several times. Uninstalled and re-installed: no success....

**Comment by jcbvm** [comment]
Can you try to remove both the `.i18n-editor-metadata` file in your project folder and the `.i18n-editor` file in your windows user folder (these files are used for storing settings, maybe there is some setting messing up the editor).
